src/NServiceBus.AcceptanceTests/Scheduling/When_scheduling_a_recurring_task.cs

```diff
@@ -1,4 +1,5 @@
-﻿namespace NServiceBus.AcceptanceTests.Scheduling
+﻿#pragma warning disable 618
```

Comment:
A few of these tests are done this way. Is this too broad? It might hide future obsoletions? Having said that, I assume these tests are probably gone in v8 where the feature is no longer supported so maybe this is OK

Reply:
Yes, these are all scheduler tests, so will be removed in v8.